utils/ixdtf/Cargo.toml

```diff
@@ -25,6 +25,7 @@ all-features = true
 [features]
 default = ["duration"]
 duration = []
+timezone = []
```

Comment:
What's the point of having this behind a feature?

Reply:
My thought was to treat it as an additional parser akin to `duration`, but I think it could be removed as well.